examples/onft721-zksync/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@layerzerolabs/onft721-zksync-example",
+  "version": "0.2.0",
```

Comment:
Since this is a new package, I think we should likely start at `0.0.1`.  This would allow us to remove the `CHANGELOG.md` and `.changeset/lovely-balloons-play.md` as it would be the initial "publish".